setup.py

```diff
@@ -40,7 +40,7 @@ def _get_plugin_version():
 
 package_name = "dbt-databricks"
 package_version = _get_plugin_version()
-dbt_core_version = "1.0.0"
+dbt_spark_version = "1.0.0"
```

Comment:
`dbt_spark_version` should be upgraded to `1.0.1rc1` or upper once `dbt-spark` bumps up the version, also in `requirements.txt`.